File: EQIS_env-main/react_sample/react/src/interfaces/file/attcFilB.ts
import { cloneDeep } from 'lodash';

import { baseInit, Ibase } from '@/interfaces/common/base';

export interface IattcFilB extends Ibase {
    // 첨부파일 DB
    attcFilNo: string;
    attcFilSeq: number;
    attcFilOgcNm: string;
    attcFilMgn: number;
    attcFilExtnNm: string;
    useYn: string;

    //Etc
    encAttcFilNo: string;
    encAttcFilSeq: string;
}

export const attcFilBInit = () => {
    const attcFilB: IattcFilB = {
        // 첨부파일 DB
        attcFilNo: '',
        attcFilSeq: 0,
        attcFilOgcNm: '',
        attcFilMgn: 0,
        attcFilExtnNm: '',
        useYn: '',
        ...baseInit(),

        //Etc
        encAttcFilNo: '',
        encAttcFilSeq: '',
    };
    return cloneDeep(attcFilB);
};

File: EQIS_env-main/react_sample/springboot/Spring/src/main/java/com/sample/basic/sample/file/model/AttcFilB.java
package com.sample.basic.sample.file.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;
import lombok.experimental.SuperBuilder;

/**
 * <pre>
 * 언어공통코드 상세
 * </pre>
 * 
 * @author user
 *
 */

@Data
@EqualsAndHashCode(callSuper = false)
@NoArgsConstructor
@AllArgsConstructor
@SuperBuilder
public class AttcFilB {
	// 첨부파일DB
	private String attcFilNo;
	private Integer attcFilSeq;
	private String attcFilOgcNm;
	private Long attcFilMgn;
	private String attcFilExtnNm;
	private String useYn;
	
	// Etc
	private String encAttcFilNo;
	private String encAttcFilSeq;
}
